pages/assessment.js

```diff
@@ -103,18 +116,42 @@ export default function Assessment({ hackathons }) {
       {hackers && (
         <>
           <Page hackathons={hackathons} currentPath="assessment">
-            <ToggleButton leftText="View All" rightText="Assigned Only" />
-            <ToolBar
-              search={setSearch}
-              reverse={setReverse}
-              sort={setSortType}
-              reversed={reverse}
-            />
-            <Table
-              displayedHackers={displayedHackers}
-              selectedHacker={selectedHacker}
-              setSelectedHacker={setSelectedHacker}
-            />
+            <ToggleBarContainer>
+              <ToggleButton leftText="Admin" rightText="Personal" isLeftSelected={isAdminView} setIsLeftSelected={setIsAdminView} />
+              <ToggleButton leftText="View All" rightText="Assigned Only" isLeftSelected={isViewAllApplicants} setIsLeftSelected={setIsViewAllApplicants} />
+            </ToggleBarContainer>
+            {isViewAllApplicants ? <span>view all applicants place holder <br /> </span> : <span>view assigned only place holder <br /> </span>}
+            {isAdminView ? 
+              <>
+              <span>Assignment page placeholder</span>
+                <ToolBar
+                  search={setSearch}
+                  reverse={setReverse}
+                  sort={setSortType}
+                  reversed={reverse}
+                />
+                <Table
+                  displayedHackers={displayedHackers}
+                  selectedHacker={selectedHacker}
+                  setSelectedHacker={setSelectedHacker}
+                />
+              </>
+              :
+              <>
+                <span>Scoring page placeholder</span>
+                <ToolBar
+                  search={setSearch}
+                  reverse={setReverse}
+                  sort={setSortType}
+                  reversed={reverse}
+                />
+                <Table
+                  displayedHackers={displayedHackers}
+                  selectedHacker={selectedHacker}
+                  setSelectedHacker={setSelectedHacker}
+                />
```

Comment:
can the ternary statement just render the different spans? since the <ToolBar> and <Table> look like they're the exact same in both cases

Reply:
yea i did that for intuition sake since that content should all be replaced by the new scoring and assignment components!